mumd/src/audio/input.rs

```diff
@@ -42,13 +44,21 @@ pub fn callback<T: Sample>(
                 .unwrap();
             opus_buf.truncate(result);
             let bytes = Bytes::copy_from_slice(&opus_buf);
-            match input_sender.try_send(VoicePacketPayload::Opus(bytes, false)) {
+            match input_sender.send(VoicePacket::Audio {
```

Comment:
Is it necessary to create the VoicePacket here or could it be created later on in the pipeline?

Reply:
The problem is the `count`, it and the actual voice data are the only non-trivial parts. Since both TCP and UDP needs to send packages and they probably want the same seq_num we might as well construct it here. It's not like we'd be sending less data.